cms/templates/js/certificate-editor.underscore

```diff
@@ -19,6 +19,10 @@
                 <textarea id="certificate-description-<%= uniqueId %>" class="certificate-description-input text input-text" name="certificate-description" placeholder="<%= gettext("Description of the certificate") %>" aria-describedby="certificate-description-<%=uniqueId %>-tip"><%= description %></textarea>
                 <span id="certificate-description-<%= uniqueId %>-tip" class="tip tip-stacked"><%= gettext("Description of the certificate") %></span>
             </div>
+            <div class="field text actual-course-title">
+                <label class="actual-course-title"><b><%= gettext("Course Title") %></b></label>
+                <span class="actual-title"><%= course.get('name') %></span>
+            </div>
```

Comment:
@mattdrayer this is the output of above change. 
![title](https://cloud.githubusercontent.com/assets/7334669/9665102/7e08de84-5287-11e5-8150-bcc749fc1b88.jpg)